### DISCUSSION TITLE
NO longer able to build with flavour

### DISCUSSION THREAD

**Discussion by RemeJuan** [codemagic-ci-cd/codemagic-docs#2498] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Trigger. build using flavours

### Expected results

Build to succeed

### Actual results

Build fails with error. `Scheme "production" not found from repository! Please reconfigure your project.`

### Build id (optional)

6621206168d504f180c49696

**Comment by dtrdic** [comment]
hey @RemeJuan looks like you have a syintax error in your build command. You have added this build argument `—flavor=production`but it should look like this `--flavor production `

![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/e188baf4-3497-4b9d-9387-8c623d3dee87)

**Reply by RemeJuan** [reply]
Thanks, but that is what it was prior, I was making changes to try and test, also why android has no flavour. Making the change suggested results in the same issue.

**Reply by dtrdic** [reply]
the above is a screenshot from my workflow configuration, not yours.
Please doublecheck the syntax you are using when adding a build argument, i can see you are still defining it like this `—flavor production ` and not like this `--flavor production `.
But also try to clean your project locally by running these commands, let's see if helps to fix your configuration issues
```
cd ios
pod cache clean --all
rm -rf ~/Library/Caches/CocoaPods
rm -rf Pods
rm -rf ~/Library/Developer/Xcode/DerivedData/*
pod deintegrate
flutter clean
pod install
```
I tested my workflow and built with flavors, everything works as expected

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.

**Comment by TheAdamGalloway** [comment]
In case anybody stumbles across this error on Google (like I did), your problem might be the capitalisation of the xcscheme files. Mine were capitalised (Production and Staging rather than production and staging). I renamed them but git didn't pick it up because Mac isn't case sensitive. I had to use `git mv ios/Runner.xcodeproj/xcshareddata/xcschemes/Production.xcscheme ios/Runner.xcodepr
oj/xcshareddata/xcschemes/production.xcscheme`
